ImperatorToCK3.UnitTests/CK3/Titles/LandedTitlesTests.cs

```diff
@@ -309,5 +311,49 @@ public class LandedTitlesTests {
 
 			Assert.Equal(20, titles["c_county1"].GetDevelopmentLevel(date)); // (10+40)/2 - sqrt(25)
 		}
+
+		[Fact]
+		public void DerivedColorsHaveSameHueAsBase() {
```

Comment:
You can probably have a more general name.

Reply:
Changing to `DerivedColorsHaveCorrectComponents`.